db/schema.rb

```diff
@@ -2,405 +2,29 @@
 # of editing this file, please use the migrations feature of Active Record to
 # incrementally modify your database, and then regenerate this schema definition.
 #
-# Note that this schema.rb definition is the authoritative source for your
-# database schema. If you need to create the application database on another
-# system, you should be using db:schema:load, not running all the migrations
-# from scratch. The latter is a flawed and unsustainable approach (the more migrations
-# you'll amass, the slower it'll run and the greater likelihood for issues).
+# This file is the source Rails uses to define your schema when running `rails
```

Comment:
Hi @cesswairimu thanks for working on this, as mentioned in the README contributing PR, we were meant to re-use the schema.rb file for the current WSO project. So maybe its not a good idea to delete stuff from this file completely? What do you think?

Reply:
I updated this PR with dev and it now seems to be failing tests due to the deletions on this files...